### DISCUSSION TITLE
Show profile picture for PLN Directory embeds

### DISCUSSION THREAD

**Discussion by AlexxNica** [memser-spaceport/pl-network-hub#9] [Ideas]
### What is the idea?

Make embeds of PLN Directory show the profile picture of teams and members.

### Why is this important?

Provide better context when sharing teams and members' profiles on messaging platforms like Slack, Discord, Telegram, etc.

### Current state of embeds

**Discord**
<img width="526" alt="image" src="https://user-images.githubusercontent.com/14338588/217426536-a86a1702-0d4f-4b58-9755-82482215536c.png">

**Slack**
<img width="541" alt="image" src="https://user-images.githubusercontent.com/14338588/217426741-e2dbefd1-50a6-4230-a0ae-8d3240daeca9.png">

**Telegram**
<img width="464" alt="image" src="https://user-images.githubusercontent.com/14338588/217426869-282aa1d9-c7be-4ab3-b5c8-18657fed2ad8.png">

**Comment by pandeyanujk** [comment]
nice to have! Thanks for adding @AlexxNica
